resources/themes/cura-light/theme.json

```diff
@@ -345,7 +345,7 @@
     },
 
     "sizes": {
-        "window_minimum_size": [70, 50],
+        "window_minimum_size": [106, 85],
```

Comment:
So people who run Cura on an XGA screen (1024x768), or even a 2560x1600 "Retina" screen (1280 x 800) are just shit out of luck?

Reply:
My mistake, the idea is to change the minimum to 1280x800. It was an internal decision made with the agreement of @Appesteijn and @LukeChen-Ultimaker.

This is how it looks currently with the minimum size (1280x800):
![image](https://user-images.githubusercontent.com/15830521/47557057-1633d580-d910-11e8-9bd0-af4f211e3c70.png)